Xbim.Ifc/IfcStore.cs

```diff
@@ -66,10 +68,18 @@ public class IfcStore : IModel, IDisposable, IFederatedModel, IEquatable<IModel>
         
         private readonly ReferencedModelCollection _referencedModels = new ReferencedModelCollection();
 
+        static IfcStore()
+        {
+
+            XbimServices.Current.ConfigureServices(s => s.AddXbimToolkit());

```

Comment:
Probably an edge case here. ConfigureServices cannot be called after the Container had been built. Need to check that.

Reply:
Fixed